packages/accessible-focus/jest.config.js

```diff
@@ -1,3 +1,3 @@
-module.exports = {
+export default {
```

Comment:
I can't find this documented on Jest website, but it works.

Inspecting the source code, it is imported by https://github.com/facebook/jest/blob/master/packages/jest-util/src/requireOrImportModule.ts. It tries to import it as CJS, and if it fails, ESM.